api/v1beta1/zz_generated.deepcopy.go

```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```

Comment:
What is your local go version? I dont think this should get removed normally

Reply:
I am on 1.16 which is recommended
Not sure why this disappears quite constantly on my computer

Reply:
Ahh, I think it's because this is added with go 1.17, which I use. We can discuss this offline then